components/lcd_touch/esp_lcd_touch_ft5x06/esp_lcd_touch_ft5x06.c

```diff
@@ -120,28 +120,28 @@ esp_err_t esp_lcd_touch_new_i2c_ft5x06(const esp_lcd_panel_io_handle_t io, const
 
     /* Prepare pin for touch interrupt */
     if (esp_lcd_touch_ft5x06->config.int_gpio_num != GPIO_NUM_NC) {
-        const gpio_config_t int_gpio_config = {
+        const gpio_config_t rst_gpio_config = {
```

Comment:
The name of variable was right.
```suggestion
        const gpio_config_t int_gpio_config = {
```

Reply:
This change wasn't fixed.